docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  mstk:
+    build: .
```

Comment:
You can tag the image (with only a single tag, but still useful) so that you can do `docker run -it mstk` outside of the folder with `docker-compose.yml`.
```suggestion
    build: .
    image: mstk:latest
```

Reply:
Good idea. I added this.